src/mono/wasm/build/WasmApp.LocalBuild.props

```
@@ -25,12 +25,11 @@
     <_TargetFrameworkForNETCoreTasks>net8.0</_TargetFrameworkForNETCoreTasks>
     <MSBuildEnableWorkloadResolver>false</MSBuildEnableWorkloadResolver>
     <SelfContained>true</SelfContained>
-    <WasmCommonTargetsPath>$(_WasmTargetsDir)</WasmCommonTargetsPath>
   </PropertyGroup>
 
   <PropertyGroup Condition="'$(RuntimeSrcDir)' != '' and '$(WasmBuildSupportDir)' == ''">
     <ArtifactsBinDir>$(RuntimeSrcDir)\artifacts\bin\</ArtifactsBinDir>
-    <EMSDK_PATH Condition="'$(EMSDK_PATH)' == ''">$(RuntimeSrcDir)\src\mono\wasm\emsdk\</EMSDK_PATH>
+    <EMSDK_PATH Condition="'$(EMSDK_PATH)' == ''">$(RuntimeSrcDir)\src\mono\browser\emsdk\</EMSDK_PATH>
```

Can we use `BrowserProjectRoot` here??

```suggestion
    <EMSDK_PATH Condition="'$(EMSDK_PATH)' == ''">$(BrowserProjectRoot)\emsdk\</EMSDK_PATH>
```

---

This is *Localbuild.props which is meant to be used outside a repo where you wouldn't have it's `Directory.Build.props` available, thus no `BrowserProjectRoot`.